### PR TITLE
doc updates

### PR DESCRIPTION
**Pull request by ocofaigh**
### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by SirSpidey — DISMISSED** [review]
👍

File: ibm_catalog.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
...
.... If user set the input variable add_service_id_creator_role to true, this module will also add \"Service ID creator\" role access for `iam-identity`. It is recommended to set it true if the service ID creation is disabled in the IAM settings.\n- Creates a new API key for the Service ID\n- Optionally creates a secrets group, if an existing one is not passed in.\n- Creates a new arbitrary secret in the Secret Group with the generated Service ID API key value. Because it is an arbitrary secret, the IAM engine does not create the key or manage its rotation.\n- Configures the Secret Manager instance with the IAM credentials engine using the Service ID API key.\n\n#### Usage\n```hcl\nmodule \"iam_secrets_engine\" {\n  # Replace \"X.X.X\" with a release version to lock into a specific release\n  source               = \"https://cm.globalcatalog.cloud.ibm.com/api/v1-beta/offering/source?archive=tgz&kind=terraform&name=terraform-ibm-secrets-manager-iam-engine&version=X.X.X\"\n  resource_group_id    = \"xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX\"\n  region               = \"us-south\"\n  iam_engine_name      = \"iam-engine\"\n  secrets_manager_guid = \"XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX\"\n}\n```",
```
Comment (rajatagarwal-ibm) [inline ibm_catalog.json]: `an IBM Secrets Manager` or `Secrets Manager`. Also can change `Secret Group` to `secret group`

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+[![Catalog release](https://img.shields.io/badge/release-IBM%20Cloud%20Catalog-3662FF?logo=ibm)](https://cloud.ibm.com/catalog/modules/terraform-ibm-secrets-manager-iam-engine-411db08f-f4fa-4b60-89e5-d9f094c5ab42-global)
 
-This module configures an [IAM secrets engine](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-configure-iam-engine) for an existing Secrets Manager instance.
+Use this module to configure a secrets manager [IAM credentials engine]((https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-configure-iam-engine)) for an existing secrets manager instance.
```
Comment (rajatagarwal-ibm) [inline README.md]: configure `an IBM Secrets Manager` or just `Secrets Manager`? The same approach should go with the - "existing secrets manager instance."

**Review by rajatagarwal-ibm — APPROVED** [review]

**Pull request merged**